### ISSUE TITLE
Document auto-upgrade setup for Docker deployments

### ISSUE THREAD

**Issue by bajtos** [filecoin-station/core#420]
Add Watchtower documentation so that Core nodes on Docker also have auto updates.

Related:
- https://github.com/filecoin-station/spark/pull/61#issuecomment-2066473226

**Comment by juliangruber** [comment]
Via Slack, needs to be tested:

```
docker run -d --name watchtower --restart=always -v /var/run/docker.sock:/var/run/docker.sock containrrr/watchtower --interval 3600 --cleanup
```

I propose we try this in core-fly, and replace the dependabot strategy?

**Comment by bajtos** [comment]
> I propose we try this in core-fly, and replace the dependabot strategy?

SGTM.

It would be even better to find a way to run Watchtower via a Docker Compose file so that people don't have to start multiple services manually. Assuming Watchtower and DockerCompose work together.

**Comment by juliangruber** [comment]
Yes! This is how Saturn does it: https://github.com/filecoin-saturn/L1-node/blob/f40048e8549063800f1fe14530d9ecfb6a1b794e/docker-compose.yml
